schemas/context/implementationdetails.schema.json

```diff
@@ -34,6 +34,18 @@
           "title": "Version",
           "type": "string",
           "description": "The version identifier of the API, e.g h.18."
+        },
+        "xdm:environment": {
+          "title": "Environment",
+          "type": "string",
+          "description": "The environment of the implementation",
+          "enum": ["web", "mobile", "server", "iot"],
```

Comment:
I wonder if we should go 'browser', 'app', 'server', 'iot'. 

It takes some of the ambiguity out since web could be on mobile or desktop. And App could be more than just mobile. It could be a native app windows or an app on Roku. 

I am on the fence for iOT since most of the time is is just an app (like a set-top box). Basically I think there is a place for a device sending data as part of the operating system (and not part of the app) which is why I think there is a place for it.

Reply:
I'll make the change.